### DISCUSSION TITLE
Alpha 1.7.1

### DISCUSSION THREAD

**Discussion by RighteousRyan1** [RighteousRyan1/WiiPlayTanksRemake#53] [General]
An update full of visual/audio polish and bug fixes.

Remember you can enable speedrun mode with `F1`! Top 5 uploaded (may be changed later) speedruns will get their spot on the main menu!

## Changes

- Made stats screen more readable and color coded.
- Chat box now shows messages from the bottom up, and text is properly scaled now.
- Made tracks more closely packed together.
- Tracks now fade at the same scale, ignoring framerate.
- Player-laid mines now activate and tick with sound cues.
- Explosions now look much more accurate to the original.
- Bullet trails (e.g: ones from the green tank) now have improved visuals.
- AI companion mode now spawns your companions in un-occupied player tank spots.
- Difficulties menu "Random Tanks" now includes support for a range of tanks.

## Fixes

- Fixed AI companion bug
- Fixed tank tracks not being placed during tank rotation
- Made infinite lives actually work on campaigns with one starting life.

## TODO

- [ ] Bonus life screen
- [ ] Overhaul the multiplayer lobby screen
- [ ] Math operations optimization
- [ ] Level Editor bug fixes (rotation bug, etc.)
	- Re-entering the level editor from testing will now no longer flip the tanks around the Y-axis.
- [ ] New AI tank aggression system (depending on the tank type, will either make advances towards you or will avoid you)

PLEASE, if you find anything wrong, go to my [discord](https://discord.gg/KhfzvbrrKx) server and report bugs in the #bugs 
channel, or, if you want to suggest something, suggest them in the #suggestions channel.

Happy tanking!

<hr /><em>This discussion was created from the release <a href='https://github.com/RighteousRyan1/TanksRebirth/releases/tag/1.7.1-alpha'>Alpha 1.7.1</a>.</em>

**Comment by damdollor99** [comment]
how do i play the game?
there is no .exe file to run.

**Reply by RighteousRyan1** [reply]
You have to download and run the executable from the zip file containing the game itself. You are installing the source code. You must install "tanks_rebirth_v1.7.1.rar" and extract it using WinRAR for the time being.

**Reply by damdollor99** [reply]
thanks
btw, i cant access my old account anymore
